ntc_templates/templates/index

```diff
@@ -674,6 +674,7 @@ huawei_smartax_display_ont_info_0_1_2.textfsm:huawei_smartax_display_ont_info_de
 huawei_smartax_display_ont_port_vlan_0_1_byport_eth_0.textfsm, .*, huawei_smartax, di[[splay]] ont p[[ort]] vl[[an]] \d+ \d+ byport eth \d+\s*
 huawei_smartax_display_ont_port_state_0_1_eth-port.textfsm, .*, huawei_smartax, di[[splay]] on[[t]] por[[t]] st[[ate]] \d+ \d+ e[[th-port]]
 huawei_smartax_display_ont_port_vlan_0_1_byvlan_0.textfsm, .*, huawei_smartax, di[[splay]] ont p[[ort]] v[[lan]] \d+ \d+ byvlan \d+
+huawei_smartax_display_location_aaaa-aaaa-aaaa.textfsm, .*, huawei_smartax, disp[[lay]] loc[[ation]] \S+\s*$
```

Comment:
```suggestion
huawei_smartax_display_location.textfsm, .*, huawei_smartax, disp[[lay]] loc[[ation]] \S+\s*$
```

Reply:
I believe that we want to keep it at the short end of the command.

Reply:
With this direction, a few other modifications need made.

I can submit a PR to Enric for the items I've worked up to get parsing to work.

- [x] Reorder `...display_location.textfsm` in the index
- [x] Rename `...display_location_aaaa-aaaa-aaaa.textfsm` template to match the index
- [x] Change test dir to `tests/huawei_smartax/display_location/`
- [x] Update `display_location` regex so that everything after the word location is optional (might not be 'most accurate', but necessary so the tests pass)

@jvanderaa Based on this list, do we want to move forward with these changes?
(Before @evilmonkey19 considers merging my PR https://github.com/evilmonkey19/ntc-templates/pull/5 into his branch?)